homeassistant/components/totalconnect/services.yaml

```diff
@@ -13,3 +13,11 @@ arm_home_instant:
     entity:
       integration: totalconnect
       domain: alarm_control_panel
+
+bypass_zone:
+  name: Bypass Zone
+  description: Bypass a security zone.
+  target:
```

Comment:
It looks like your service requires a zone_id parameter, that should be defined here as a field. https://developers.home-assistant.io/docs/dev_101_services/ shows an example of how to do this.

Reply:
missed that, thanks